recoleta-tj/months.txt

```diff
@@ -0,0 +1,12 @@
+1
```

Comment:
Não precisamos disso num arquivo, certo? Pode ser um for no script go ou uma variável de ambiente se quiser deixar configurável. Inclusive, já fazemos isso nos outros scripts.

Mesma coisa para os anos.